vowel.js

```diff
@@ -3,28 +3,32 @@ function countVowels(paragraph){
     var counter = 0;
     var characterCounter={}
     
-    var set1= new Set()
+    var vowelSet= new Set()
     var vowels=['a','e','i','o','u'] 
     
     for(var i=0;i<=paragraph.length;i++){
     if(vowels.includes(paragraph.charAt(i))){
-    set1.add(paragraph.charAt(i))
+    vowelSet.add(paragraph.charAt(i))
     }
     if(!characterCounter.hasOwnProperty(paragraph.charAt(i))){
     characterCounter[paragraph.charAt(i)] = 1
     //characterCounter[i]: myValue
     //add it as key to my bject with value of 1
     }
-    else if(characterCounter[(paragraph.charAt(i))] == 1){
-     counter =counter+ 1 
-    characterCounter[paragraph.charAt(i)] =characterCounter[paragraph.charAt(i)] +1
-    
+    else{ 
+ 	if(characterCounter[(paragraph.charAt(i))] == 1){
+  
+ counter =counter+ 1
+     }
+    characterCounter[paragraph.charAt(i)] =characterCounter[paragraph.charAt(i)] +1  
     }
     }
-    
-    newVowel= Array.from(set1).join('')
+      console.log(characterCounter)
```

Comment:
I suppose you need this `console.log` for debugging purposes, you should remove it when you are done using it.